jet/urls.py

```diff
@@ -24,13 +24,23 @@
         model_lookup_view,
         name='model_lookup'
     ),
-    url(
+]
+
+if django.VERSION[0] >= 2:
+    from django.views.i18n import JavascriptCatalog
```

Comment:
I have a typo error with the django 2.0 version I think it should be "JavaScriptCatalog"